app/src/main/java/com/group80/uoftinder/firebase/realtime/UserRealtimeDbFacade.java

```diff
@@ -0,0 +1,77 @@
+package com.group80.uoftinder.firebase.realtime;
+
+import com.group80.uoftinder.entities.User;
+
+import java.util.List;
+
+/**
+ * A Facade for methods regrading User in realtime database. All methods are constants, and no
+ * instances of this Facade should be created.
+ */
+public class UserRealtimeDbFacade {
+    private static final String DEFAULT_USER_TYPE = "academic";
```

Comment:
* See [Java 11 API - Enum](https://docs.oracle.com/en/java/javase/11/docs/api/) for the use of `Enum`
* See [here](https://github.com/CSC207-2022F-UofT/course-project-group-80/pull/25/files#r1027176005) for how to use `Enum` in our case. 

Basically, by haveing `toString()` in our `Enum`, we can have similar behaviour as Strings. Idealy we should have `UserType.ACADEMIC` with the type being `UserType`, but considering everything is in terms of `String` for now, I have adapted `String`. Since Java is smart enough to automatically call `UserType.toString()`, we shouldn't have issues migrating to `Enum` later. 

```java
private void doStuff(String userType) {}; // a method we have 
...
doStuff("Academic"); // OK
doStuff(UserType.ACADEMIC); // OK
```